config/purposes/pWGS_384_bottom.yml

```diff
@@ -0,0 +1,26 @@
+---
+pWGS-384 Post Shear XP:
+  :asset_type: plate
+  :creator_class: LabwareCreators::QuadrantStamp
+  :merger_plate: true
+  :size: 384
+pWGS-384 End Prep:
+  :asset_type: plate
+  :presenter_class: Presenters::PermissivePresenter
+  :size: 384
+pWGS-384 AL Lib:
+  :asset_type: plate
+  :size: 384
+pWGS-384 Lib PCR:
+  :asset_type: plate
+  :size: 384
+  :creator_class: LabwareCreators::TaggedPlate
+  :tag_layout_templates:
+  - 'TSsc384'
```

Comment:
The most similar purpose seems to be: LHR-384 Lib PCR. That one is using 
```yaml
  :presenter_class: Presenters::MinimalPcrPlatePresenter
```
That class does not have documentation, but naming suggests it could be related with this type of plate. Did you check this class already?

Reply:
As suggested by the name, this class seems to reduce the amount of information displayed in the UI. Given that this presentation style is used for most other, if not all, PCR plates, I will adopt it for the pWGS-384 pipeline too